bare_metal_billing/billing.py

```diff
@@ -0,0 +1,118 @@
+import csv
+import math
+from datetime import datetime
+from dataclasses import dataclass
+
+from bare_metal_billing import models
+
+
+BM_SU_LIST = [
+    "BM FC430",
+    "BM FC830",
+    "BM GPUA100SXM4",
+    "BM GPUH100",
+]
+
+RESOURCE_CLASS_2_SU_MAPPING = {
+    "lenovo-sd665nv3-h100": "BM GPUH100",
+    "lenovo-sd650nv2-a100": "BM GPUA100SXM4",
+    "sd650nv2": "BM GPUA100SXM4",
+    "fc430": "BM FC430",
+    "fc830": "BM FC830",
+}
```

Comment:
There are two other resource classes that might be relevant:

`r740xd_amdmi210`: node with an AMD Aldebaran/MI200 [Instinct MI210] GPU; there's only one, and it's leased to Triton
`fc830-nvme`: fc830 configured with nvme storage; there are 18, and all are leased to the storage-research project

Reply:
@joachimweyl Can you confirm how these should be handled in billing?

Reply:
@QuanMPhm Should this be read from a config file (json, yaml) - BM_SU_LIST and the mapping?

Reply:
for now ignore these. We do not have a pricing for them. I will keep a list of these and bring them up with the admin team to discuss how to bill for them eventually. I guess set pricing to 0 until further notice so they still exist in our records.

Reply:
Thanks for confirming all this!